File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/EventSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class EventSearch extends AbstractEntity {
	String eventName;
	String eventDescription;
	@Temporal(TemporalType.DATE)
	Date eventStartDate;
	@Temporal(TemporalType.DATE)
	Date eventEndDate;
}

File: scholar-web/src/main/webapp/app/model/staff/event/EventSearch.js
Ext.define('scholar.model.staff.event.EventSearch', {
	extend: 'Ext.data.Model',
	fields : [ {
		name : 'id',
		type : 'int'
	}, {
		name : 'eventName',
		type : 'string'
	}, {
		name : 'eventDescription',
		type : 'string'
	},
	{
		name : 'eventStartDate',
		type : 'date',
		dateFormat : 'n/j h:ia'
	},
	{
		name : 'eventEndDate',
		type : 'date',
		dateFormat : 'n/j h:ia'
	}],
	
});